graphql/schemabuilder/pagination.go

```diff
@@ -389,95 +350,63 @@ func (sb *schemaBuilder) buildPaginatedField(typ reflect.Type, f interface{}) (*
 		return nil, fmt.Errorf("a key field should be registered on the return object")
```

Comment:
maybe mention pagination in this error, i.e. "a key field must be registered for paginated objects"

Reply:
okay